tests/test_importers.py

```diff
@@ -1,86 +1,82 @@
-            rtol=RTOL, atol=ATOL)
+                    rtol=RTOL, atol=ATOL)
     assert_allclose(gas_estimated.values, [False, False, True, False],
-            rtol=RTOL, atol=ATOL)
+                    rtol=RTOL, atol=ATOL)
+
 
 def test_import_green_button_xml(consumption_gbxml_filename):
     cd = import_green_button_xml(consumption_gbxml_filename)
 
     assert_allclose(cd.data.values,
-            [25.662, 21.021, 21.021, 21.021, 21.021,
-             21.021, 25.662, 25.662, 21.021, 21.021,
-             21.021, 21.021, 21.021, 25.662, 25.662,
-             21.021, 21.021, 21.021, 21.021, 21.021,
-             25.662, 25.662, 21.021, 21.021, 21.021,
-             21.021, 21.021, 25.662, 25.662, 21.021,
-             21.021, np.nan, 25.662, 25.662, 21.021,
-             21.021, 21.021, 21.021, 21.021, 25.662,
-             25.389, 21.021, 21.021, 21.021, 21.021,
-             21.021, 25.662, 25.662, 21.021, 21.021,
-             21.021, 21.021, np.nan], rtol=RTOL, atol=ATOL)
+                    [25.662, 21.021, 21.021, 21.021, 21.021,
+                     21.021, 25.662, 25.662, 21.021, 21.021,
+                     21.021, 21.021, 21.021, 25.662, 25.662,
+                     21.021, 21.021, 21.021, 21.021, 21.021,
+                     25.662, 25.662, 21.021, 21.021, 21.021,
+                     21.021, 21.021, 25.662, 25.662, 21.021,
+                     21.021, np.nan, 25.662, 25.662, 21.021,
+                     21.021, 21.021, 21.021, 21.021, 25.662,
+                     25.389, 21.021, 21.021, 21.021, 21.021,
+                     21.021, 25.662, 25.662, 21.021, 21.021,
+                     21.021, 21.021, np.nan], rtol=RTOL, atol=ATOL)
+
 
 def test_import_seed_timeseries():
     fd, fname = tempfile.mkstemp()
     db_url = 'sqlite:///{}'.format(fname)
     engine = create_engine(db_url, echo=True)
     metadata = MetaData()
     seed_meter = Table('seed_meter', metadata,
-        Column('id', Integer, primary_key=True),
-        Column('name', String),
-        Column('energy_type', Integer),
-        Column('energy_units', Integer),
-    )
-    seed_timeseries = Table('seed_timeseries', metadata,
-        Column('id', Integer, primary_key=True),
-        Column('meter_id', None, ForeignKey('seed_meter.id')),
-        Column('reading', Float),
-        Column('cost', Numeric),
-        Column('begin_time', TIMESTAMP),
-        Column('end_time', TIMESTAMP),
-    )
-    seed_meter_building_snapshot = Table('seed_meter_building_snapshot', metadata,
-        Column('id', Integer, primary_key=True),
-        Column('meter_id', None, ForeignKey('seed_meter.id')),
-        Column('buildingsnapshot_id', Integer),
-    )
+                       Column('id', Integer, primary_key=True),
+                       Column('name', String),
+                       Column('energy_type', Integer),
+                       Column('energy_units', Integer),
+                       )
+    seed_timeseries = Table(
```

Comment:
Funky `autopep8` issue. This importer will be removed in v0.4 (v0.4 will not be backwards compatible and as far as I know, no one is using this feature), so no real reason to fix.